awspub/image.py

```diff
@@ -556,6 +557,18 @@ def publish(self) -> None:
                     f"currently using partition {partition}. Ignoring marketplace config."
                 )
 
+        # send ssn notification
+        if self.conf["sns"]:
+            for region in self.image_regions:
+                ec2client = boto3.client("ec2", region_name=region)
+                image_info = self._get(ec2client)
+                if image_info:
```

Comment:
nit: can you do:
```
if not image_info:
    logger.error(f"can not send SNS notification for {self.image_name} because no image found in {region}" )
    continue
    
SNSNotification(self.conf, image_info.image_id, region_name=region).publish()
```

that matches the style in ssm

Reply:
changed to match ssm style